react/events/enhancedCommerce.ts

```diff
@@ -92,6 +93,40 @@ export const productImpression = (
   })
 }
 
+export const addToCart = (product: Product, price: number, quantity: number) => {
```

Comment:
Add the documentation https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce#add-remove-cart